Fachportal Padagogik.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "4b0b42df-76b7-4a61-91aa-b15bc553b77d",
 	"label": "Fachportal Pädagogik",
 	"creator": "Philipp Zumstein",
-	"target": "^https?://www\\.fachportal-paedagogik\\.de/fis_bildung/",
+	"target": "^https?://www\\.fachportal-paedagogik\\.de/fis_bildung/|^https?://www?\\.pedocs\\.de/",
```

Comment:
are you sure you want `www?`? (looks like anything but `www.` is forbidden)

But in general, I would prefer `^https?://(www\\.fachportal-paedagogik\\.de/fis_bildung/|www\\.pedocs\\.de/)`